vendor/bat-native-ads/src/bat/ads/internal/account/account.cc

```diff
@@ -66,19 +66,19 @@ void Account::RemoveObserver(AccountObserver* observer) {
 }
 
 bool Account::SetWallet(const std::string& id, const std::string& seed) {
-  const WalletInfo last_wallet = wallet_->Get();
+  const WalletInfo& last_wallet = wallet_->Get();
```

Comment:
In this case adding a reference `&` dosn't do any optimizations because wallet_->Get() returns a copy of WalletInfo object. 
I think `const WalletInfo last_wallet = wallet_->Get();` is better here.

Reply:
Agreed to followup in a separate issue